docs/cc_deps.md

```diff
@@ -0,0 +1,27 @@
+<a name="#cc_fuzz_test"></a>
```

Comment:
Tengpeng, I believe the standard is to include API documentation directly in the source code, as Starlark docstrings.

Reply:
Just to clarify, do you mean we don't need `cc_deps.md` and `common.md` to list those rules, instead, just the docstrings in the source code?

Reply:
When included, recommend specifying what `corpus` expects as srcs